shell/platform/glfw/flutter_glfw.cc

```diff
@@ -531,6 +531,14 @@ static FLUTTER_API_SYMBOL(FlutterEngine)
   args.command_line_argv = &argv[0];
   args.platform_message_callback = GLFWOnFlutterPlatformMessage;
   args.custom_task_runners = custom_task_runners;
+  // Provide path to .so in AOT configurations.
+  std::string shared_library_path = std::string();
+  shared_library_path.append(engine_properties.assets_path);
+  shared_library_path.append("/libapp.so");
```

Comment:
This isn't going to work on Windows.

Reply:
I don't believe we use glfw on windows anymore, @stuartmorgan to clarify

Reply:
Theoretically the GLFW embedding can be built for any of the desktop platforms; in practice for the ones where we have a native embedding the support is minimal (e.g., we don't even build it on bots for macOS I don't think).

I'd rather we not actively break things, but I'm not especially concerned if new things don't work on platforms where we're not using it. (Eventually we should either wire up tests on all three platforms, or rip it out where it's not the sole option, but for now it's just best-effort.)